docs/discover/document-explorer.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[document-explorer]]
+== Document Explorer
+
+beta::[]
+
+*Discover* has a new *Document Explorer* with resizable columns, better data sorting and comparison,
```

Comment:
This sentence is very 8.0.0-specific. Are you planning to update this for 8.1.0? Maybe we keep `new` things in the release docs?